lti_consumer/lti_xblock.py

```diff
@@ -1506,17 +1572,23 @@ def _get_context_for_template(self):
         Returns:
             dict: Context variables for templates
         """
-
         # For more context on ALLOWED_TAGS and ALLOWED_ATTRIBUTES
         # Look into this documentation URL see https://bleach.readthedocs.io/en/latest/clean.html#allowed-tags-tags
         # This lets all plaintext through.
         allowed_tags = bleach.sanitizer.ALLOWED_TAGS + ['img']
         allowed_attributes = dict(bleach.sanitizer.ALLOWED_ATTRIBUTES, **{'img': ['src', 'alt']})
         sanitized_comment = bleach.clean(self.score_comment, tags=allowed_tags, attributes=allowed_attributes)
+
         launch_url = self.launch_url
+        lti_1p3_launch_url = self.lti_1p3_launch_url.strip()
+
+        lti_consumer = None
+        if self.config_type in ("external", "database"):
+            lti_consumer = self._get_lti_consumer()
 
-        if self.config_type == "external":
-            launch_url = self._get_lti_consumer().lti_launch_url
+        # The lti_launch_url property only exists on the LtiConsumer1p1.
```

Comment:
if config is external here we will set launch_url no matter what the version is in the boolean logic below - either this comment should change or the below should be config in (external, database) AND version = 1.1

Reply:
You're right. We don't support the external config for LTI 1.3, so I must have been relying on that, but I don't like that implicit condition. I'll make it explicit. Thanks!